### ISSUE TITLE
Icon missing in Firefox 43

### ISSUE THREAD

**Issue by yan12125** [whuhacker/Unblock-Youku-Firefox#17]
As following:

![screenshot_2015-12-21_12-38-31](https://cloud.githubusercontent.com/assets/1937689/11923736/b8631d14-a7e2-11e5-9e5f-03688232ec56.png)

Seems the official addon is packaged by jpm? mozilla-jetpack/jpm#197 states that using relative URLs for icons does not work.

**Comment by whuhacker** [comment]
Thanks for your report.
Yes, the addon is packaged by jpm since version 1.0
Hope the bug in jpm be fixed soon

**Comment by yan12125** [comment]
jpm 1.1.0 is released and the bug should be fixed. Could you upload a new version?

**Comment by yan12125** [comment]
Both jpm and legacy addons are deprecated, and the [relevant Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1141839) is closed as WONTFIX. I think all we need is moving to WebExtensions. (https://github.com/uku/Unblock-Youku/issues/792)
